code/modules/clothing/suits/armor.dm

```diff
@@ -441,3 +441,21 @@
 	src.icon_state = "reactiveoff"
 	src.item_state = "reactiveoff"
 	..()
+
+/obj/item/clothing/suit/armor/crusader
+	name = "crusader armor"
+	desc = "God will protect those who belive."
+	icon_state = "crusader_suit"
+	item_state = "crusader_suit"
+	slowdown = 0.3
+	matter = list(MATERIAL_BIOMATTER = 25, MATERIAL_PLASTEEL = 10, MATERIAL_STEEL = 15, MATERIAL_GOLD = 2)
+	armor = list(
+		melee = 70,
+		bullet = 50,
+		energy = 30,
+		bomb = 0,
```

Comment:
Since churchies have an assortment of grenades, let's give this armor a little bit of bomb protection. Since it's a full set of armor and has a lot of padding, we'll give it a decent amount of bomb protection.
```suggestion
		bomb = 30,
```

Reply:
Done.